cinema/serializers.py

```diff
@@ -1 +1,102 @@
-# write serializers here
+from rest_framework import serializers
+from .models import Genre, Movie, Actor, MovieSession, CinemaHall
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.CharField(max_length=514, read_only=True)
+
+    class Meta:
+        model = Actor
+        fields = "__all__"
```

Comment:
what's the difference? It's just different styling, idk, I either duplicate field names, in meta class, which is good for ordering, or explicitly define it in class as a field, and then use it with `__all__` so we can test more easily, I mean we see, where we use all default fields and django is responding for them and we see explicitly, which field we are adding to model serializer, like this precomputed field, and I just like second style more.

About seconds approach, great idea, But problem is not in defining it, but in described task and tests. We could simply use any useful approach, like adding string related field to movie, so we can see what we need to see, since it's not even desribed that we need that field in Actor class, according to readme, and I jsut was guessing what exactly do we need, and seen in traceback, that there was need in providing field full_name for and actor(I seen this in movie, not in actor, first). So I just hadn't ideas what I could use and what not, and that was the biggest problem, thanks for clarifying how I could make it an pass the test...